File: app/src/main/java/com/mikhailovskii/trakttv/ui/base/BaseFragment.java
package com.mikhailovskii.trakttv.ui.base;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public abstract class BaseFragment extends Fragment {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setHasOptionsMenu(true);
    }

}

File: app/src/main/java/com/mikhailovskii/trakttv/ui/base/BaseFragment.kt
package com.mikhailovskii.trakttv.ui.base

import android.os.Bundle
import androidx.fragment.app.Fragment

abstract class BaseFragment : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setHasOptionsMenu(true)
    }

}
